cppman/main.py

```diff
@@ -61,6 +61,16 @@ def extract_name(self, data):
         name = re.sub(r'&lt;', r'<', name)
         return name
 
+    def extract_std(self, data):
+        """Extract standard from web page."""
+        std = re.search('<h1[^>]*>(.+?)</h1>', data).group(1)
+        # cplusplus.com
+        if re.search('<span[^>]*cpp11warning[^>]*>', std):
+            std = "c++11"
```

Comment:
What about C++14 and above?

Reply:
cplusplus.com does not have C++14 and above....

Reply:
Can you add support for cppreference.com?